141. Linked List Cycle.py

```diff
@@ -14,5 +14,6 @@ def hasCycle(self, head: Optional[ListNode]) -> bool:
             else:
                 head.val=flag
             head=head.next
+            
         return False
         
```

Comment:
It would be good if you could comment explaning on what flag is and why you set it to such a number.